pkg/runtime/runtime.go

```diff
@@ -94,6 +94,7 @@ const (
 	bindingsConcurrencyParallel   = "parallel"
 	bindingsConcurrencySequential = "sequential"
 	pubsubName                    = "pubsubName"
+	deadLetterKeyFormat           = "%s||%s" // componentName||topicName
```

Comment:
You might have two different components with the same name, deployed to different namespaces:

mypubsub deployed to ns1, and mypubsub deployed to ns2.
Does this need to be changed to include a namespace?

Reply:
Yes, I'll fix it.

Reply:
After a view of the code, components in the different namespaces will be ignored. https://github.com/dapr/dapr/blob/master/pkg/runtime/runtime.go#L1754